### PR TITLE
Enhacement home page.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/app/issues/15

### CODE REVIEW

File: resources/layout/_footer.php [hunk 1]
```diff
@@ -1,13 +1,37 @@
 <div class="columns is-mobile">
     <div class="column has-text-left has-text-light">
-        Left
+        <i class="fas fa-copyright fa-inverse is-hidden-mobile"></i>
+        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank">
+            YiiFramework - <?php echo date('Y') ?>
```
Comment (roxblnfk) [inline resources/layout/_footer.php:5]:
```suggestion
            YiiFramework - <?= date('Y') ?>
```

File: resources/layout/_footer.php [hunk 2]
```diff
@@ -1,13 +1,37 @@
 <div class="columns is-mobile">
     <div class="column has-text-left has-text-light">
-        Left
+        <i class="fas fa-copyright fa-inverse is-hidden-mobile"></i>
+        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:4]:
```suggestion
        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
```

File: resources/layout/_footer.php [hunk 3]
```diff
@@ -1,13 +1,37 @@
...
+            YiiFramework - <?php echo date('Y') ?>
+        </a>
+        <a class="is-hidden-desktop" href="https://www.yiiframework.com/" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:7]:
```suggestion
        <a class="is-hidden-desktop" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
```

File: resources/layout/_footer.php [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
     </div>
 
     <div class="column has-text-centered has-text-light">
-        Center
     </div>
 
     <div class="column has-text-right has-text-light">
-        Right
+        <span class="icon">
+            <a href="https://github.com/yiisoft" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:17]:
```suggestion
            <a href="https://github.com/yiisoft" target="_blank" rel="noopener">
```

File: resources/layout/_footer.php [hunk 5]
```diff
@@ -1,13 +1,37 @@
...
+                <i class="fab fa-github fa-inverse" aria-hidden="true"></i>
+            </a>
+        </span>
+        <span class="icon is-hidden-mobile">
+            <a href="https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:22]:
```suggestion
            <a href="https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA" target="_blank" rel="noopener">
```

File: resources/layout/_footer.php [hunk 6]
```diff
@@ -1,13 +1,37 @@
...
+                <i class="fab fa-slack fa-inverse " aria-hidden="true"></i>
+            </a>
+        </span>
+        <span class="icon">
+            <a href="https://www.facebook.com/groups/yiitalk" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:27]:
```suggestion
            <a href="https://www.facebook.com/groups/yiitalk" target="_blank" rel="noopener">
```

File: resources/layout/_footer.php [hunk 7]
```diff
@@ -1,13 +1,37 @@
...
+                <i class="fab fa-facebook-f fa-inverse" aria-hidden="true"></i>
+            </a>
+        </span>
+        <span class="icon">
+            <a href="https://twitter.com/yiiframework" target="_blank">
```
Comment (roxblnfk) [inline resources/layout/_footer.php:32]:
```suggestion
            <a href="https://twitter.com/yiiframework" target="_blank" rel="noopener">
```

File: resources/views/site/index.php [hunk 8]
```diff
@@ -12,3 +12,7 @@
 <h1 class="title">Hello World</h1>
 
 <p class="subtitle">My first website with <strong>Yii 3.0</strong>!</p>
+
+<a href="https://github.com/yiisoft/docs/tree/master/guide/en" target="_blank">
```
Comment (roxblnfk) [inline resources/views/site/index.php:16]:
```suggestion
<a href="https://github.com/yiisoft/docs/tree/master/guide/en" target="_blank" rel="noopener">
```

File: resources/layout/main.php [hunk 9]
```diff
@@ -17,6 +18,7 @@
 
 $assetManager->register([
     AppAsset::class,
+    CdnFontAwesomeAsset::class
```
Comment (roxblnfk) [inline resources/layout/main.php:21]:
```suggestion
    CdnFontAwesomeAsset::class,
```

File: resources/layout/_footer.php (hunk 1 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:5]: done.

File: resources/layout/_footer.php (hunk 2 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:4]: Done.

File: resources/layout/_footer.php (hunk 3 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:7]: Done.

File: resources/layout/_footer.php (hunk 5 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:22]: Done.

File: resources/layout/_footer.php (hunk 6 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:27]: Done.

File: resources/layout/_footer.php (hunk 7 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:32]: Done.

File: resources/views/site/index.php (hunk 8 above)
Comment (terabytesoftw) [reply resources/views/site/index.php:16]: Done.

File: resources/layout/_footer.php (hunk 4 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:17]: Done.

File: resources/layout/main.php (hunk 9 above)
Comment (terabytesoftw) [reply resources/layout/main.php]: Done.

File: resources/layout/_footer.php [hunk 10]
```diff
@@ -1,13 +1,37 @@
 <div class="columns is-mobile">
     <div class="column has-text-left has-text-light">
-        Left
+        <i class="fas fa-copyright fa-inverse is-hidden-mobile"></i>
+        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
+            YiiFramework - <?= date('Y') ?>
```
Comment (xepozz) [inline resources/layout/_footer.php:5]:
```suggestion
            Yii Framework - <?= date('Y') ?>
```

File: resources/layout/_footer.php [hunk 11]
```diff
@@ -1,13 +1,37 @@
...
+        </a>
+        <a class="is-hidden-desktop" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
+            YiiFramework
```
Comment (xepozz) [inline resources/layout/_footer.php:8]:
```suggestion
            Yii Framework
```

File: resources/layout/_footer.php [hunk 12]
```diff
@@ -1,13 +1,37 @@
 <div class="columns is-mobile">
     <div class="column has-text-left has-text-light">
-        Left
+        <i class="fas fa-copyright fa-inverse is-hidden-mobile"></i>
+        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
```
Comment (xepozz) [inline resources/layout/_footer.php:4]: Why this trick is needed?

File: resources/layout/_footer.php [hunk 13]
```diff
@@ -1,13 +1,37 @@
...
+        <span class="icon">
+            <a href="https://www.facebook.com/groups/yiitalk" target="_blank" rel="noopener">
+                <i class="fab fa-facebook-f fa-inverse" aria-hidden="true"></i>
+            </a>
+        </span>
+        <span class="icon">
+            <a href="https://twitter.com/yiiframework" target="_blank" rel="noopener">
+                <i class="fab fa-twitter fa-inverse" aria-hidden="true"></i>
+            </a>
+        </span>
```
Comment (xepozz) [inline resources/layout/_footer.php:35]: What about to add telegram? :)

File: resources/layout/_footer.php (hunk 12 above)
Comment (samdark) [reply resources/layout/_footer.php:4]: `target="_blank" rel="noopener"`? Security concerns: https://web.dev/external-anchors-use-rel-noopener/

**Comment by samdark** [comment]
@terabytesoftw let's fix the rest of comments and merge it. Good changes 👍

File: resources/layout/_footer.php (hunk 10 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php]: done.

File: resources/layout/_footer.php (hunk 11 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php]: Done.

File: resources/layout/_footer.php (hunk 13 above)
Comment (terabytesoftw) [reply resources/layout/_footer.php:35]: done.

**Pull request merged**